src/main/java/org/maxgamer/quickshop/Shop/VirtualDisplayItem.java

```diff
@@ -1,3 +1,21 @@
+/*
+ * This file is a part of project QuickShop, the name is BootError.java
```

Comment:
The header created from template, if you do it by your self, you must rename the file name.